src/index.js

```diff
@@ -5,6 +5,20 @@ import Results from './Results';
 import Observable from './observable';
 import { getControlValue } from './utils';
 
+//mic constants
+const MIC_STATUS = {
+  inactive: 'INACTIVE',
+  stopped: 'STOPPED',
+  active: 'ACTIVE',
+  denied: 'DENIED'
+};
```

Comment:
Can we remove it?
Since we're using flow for types, it'll throw an error in case of invalid assignment.

Reply:
kept these constants, so that whenever the need arises to change the status, need not change it in the whole code. We can just change here.